src/main/java/com/nekromant/telegram/service/ResumeAnalysisRequestService.java

```diff
@@ -1,17 +1,95 @@
 package com.nekromant.telegram.service;
 
+import com.google.gson.Gson;
+import com.nekromant.telegram.MentoringReviewBot;
+import com.nekromant.telegram.commands.dto.ChequeDTO;
+import com.nekromant.telegram.commands.dto.LifePayResponseDTO;
+import com.nekromant.telegram.commands.feign.LifePayFeign;
+import com.nekromant.telegram.commands.feign.TelegramFeign;
+import com.nekromant.telegram.config.CVAnalProp;
+import com.nekromant.telegram.contants.PayStatus;
+import com.nekromant.telegram.model.PaymentDetails;
 import com.nekromant.telegram.model.ResumeAnalysisRequest;
+import com.nekromant.telegram.repository.PaymentDetailsRepository;
 import com.nekromant.telegram.repository.ResumeAnalysisRequestRepository;
+import feign.form.FormData;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
+import javax.ws.rs.core.MediaType;
+
+import static java.rmi.server.LogStream.log;
+
+
+@Slf4j
 @Service
 public class ResumeAnalysisRequestService {
     @Autowired
     private ResumeAnalysisRequestRepository resumeAnalysisRequestRepository;
+    @Autowired
+    private PaymentDetailsRepository paymentDetailsRepository;
+    @Autowired
+    private UserInfoService userInfoService;
+    @Autowired
+    private MentoringReviewBot mentoringReviewBot;
+    @Autowired
+    private LifePayFeign lifePayFeign;
+    @Autowired
+    private TelegramFeign telegramFeign;
+    @Autowired
+    private CVAnalProp props;
```

Comment:
можно мне пожалуйста платить без каких-то анальных пропов? 🤡 
```suggestion
    private CVAnalysisProp props;
```

Reply:
уже исправлено)